src/Compilers/Core/Portable/Operations/OperationInterfaces.xml

```diff
@@ -3366,4 +3366,25 @@
       </Comments>
     </Property>
   </Node>
+  <Node Name="IInlineArrayAccessOperation" Base="IOperation" ChildrenOrder="Instance,Argument" HasType="true">
```

Comment:
I'm not sure why we couldn't simply reuse `IArrayElementReferenceOperation` for this? And even if we don't, we should probably keep the naming consistent.

Reply:
> I'm not sure why we couldn't simply reuse ```IArrayElementReferenceOperation``` for this?

I will be happy to have this discussion during API review meeting. Personally, I don't feel that reusing is appropriate, given that the node doesn't always represent an element access.

> And even if we don't, we should probably keep the naming consistent.

I think that there is more similarities with ```IImplicitIndexerReferenceOperation``` and I kept the naming consistent with that node. Again, we will review the shape of the API later.